.stylelintrc

```diff
@@ -4,13 +4,14 @@
     "stylelint-config-palantir/sass.js"
   ],
   "rules": {
+    "declaration-empty-line-before": null,
```

Comment:
this new rule is nice but would require sweeping changes to support:
either adding empty lines before decls, or removing the handful that are there. also removing empty lines between property groups, which are helpful when we use them.